app/validators/postcode_validator.rb

```diff
@@ -1,9 +1,10 @@
 class PostcodeValidator < ActiveModel::Validator
   VALID_POSTCODE = /^([A-Z]{1,2}[0-9][A-Z0-9]? ?[0-9][A-Z]{2})$/i.freeze
   INVALID_POSTCODE = /^(BF|BX|GIR|XX|GY|JE|IM|AI|GX|KY|VG)/i.freeze
+  DISALLOWED_CHARS = /[`~,.<>;':"\/\[\]|{}()=_+-]| /.freeze
```

Comment:
Could we use `\s` instead of a space here?  This would pick up other whitespace characters that aren't spaces.

```suggestion
  DISALLOWED_CHARS = /[`~,.<>;':"\/\[\]|{}()=_+-]|\s/.freeze
```